docs/spelling_wordlist.txt

```diff
@@ -204,4 +205,4 @@ serverless
 cattrs
 IAST
 DES
-Blowfish
\ No newline at end of file
+Blowfish
```

Comment:
Is this a weird EOF issue?

Reply:
yeah I think so, I made the same edit to all 3 branches, seems like something changed in 1.7 🤷‍♂️

Reply:
Weird, I don't see how git marked this as a change...